docs/guides/color.md

```diff
@@ -0,0 +1,320 @@
+(color)=
+
+# Colors and colormaps
+
+This document describes how to programmatically define colors and colormaps in napari.
+It is intended for users of napari that are comfortable using the napari API and
+developers who want to include color as part of something they are building.
+
+
+## Colors
+
+In many places, napari allows the specification of color.
+This might be the color of an individual point, the color of text annotations, or a color value in a colormap.
+For example, the face color of all points in a layer can be defined as red.
+
+```python
+In [1]: points.face_color = 'red'
+```
+
+### Standard form
+
+Internally, napari typically coerces an input color value like `'red'` to an RGBA (red, green, blue, alpha)
+array with `shape=(4,)` and `dtype=np.float32` where the values are in `[0, 1]`.
+Each element of this array defines the contribution of each RGBA color component to the overall color,
+where a value of 0 means there is no contribution and a value of 1 means that component is saturated.
+
+For example, a purely `'red'` color is coerced to the array `[1, 0, 0, 1]` where the red component is
+saturated, but the green and blue components do not contribute at all.
+The last element (alpha) similarly describes the opacity of the color, where a value of 0 corresponds to
+transparent and a value of 1 corresponds to opaque.
```

Comment:
I would add here that in general omitting `alpha` is equivalent to setting it to `1`.